### DISCUSSION TITLE
Checkboxes in a group

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [zauberzeug/nicegui#2628] [Q&A] [answered]
### Question

Are checkboxes in a group supported - specifically, 2 checkboxes where:

both checkboxes can be off
only one can be on
turn on to off 
but doesn't toggle ie. clicking an on doesn't turn it off and the other on

Hope this makes some sense :)

**Comment by falkoschindler** [comment]
Hi @dineshbvadhia,

What about simply using multiple `ui.checkbox` elements?
```py
with ui.card():
    ui.checkbox('A')
    ui.checkbox('B')
    ui.checkbox('C')
```

**Reply by dineshbvadhia** [reply, marked as answer]
Ok. This is what I have in a non-nicegui app and so good to know :)
